connectors/migrations/20241218_backfill_notion_unkown.ts

```diff
@@ -0,0 +1,35 @@
+import { makeScript } from "scripts/helpers";
+
+import { dataSourceConfigFromConnector } from "@connectors/lib/api/data_source_config";
+import { concurrentExecutor } from "@connectors/lib/async_utils";
+import { upsertDataSourceFolder } from "@connectors/lib/data_sources";
+import { ConnectorResource } from "@connectors/resources/connector_resource";
+
+makeScript({}, async ({ execute }, logger) => {
+  const connectors = await ConnectorResource.listByType("notion", {});
+
+  await concurrentExecutor(
+    connectors,
+    async (connector) => {
+      const folderId = `notion-unknown`;
+      if (execute) {
+        await upsertDataSourceFolder({
```

Comment:
could we reuse the function `upsertNotionUnknownFolder`?

Reply:
I thought I'd use it in the backfill but since I don't I'll inline it, there's really no point in having a function here

Reply:
sorry misread that

Reply:
I don't reuse it because folderId is defined above for dry-run logging purposes